### PR TITLE
Actions merge

### PR DESCRIPTION
**Pull request by marsninja**
Now uses a single decorators for both local actions and remote actions, everything should work as normal.

### CODE REVIEW

File: jaseci_core/jaseci/actions/live_actions.py [hunk 1]
```diff
@@ -4,21 +4,29 @@
...
 live_actions = {}
 
 
-def jaseci_action(act_group=None):
+def jaseci_action(act_group=None, aliases=list(), allow_remote=False):
     """Decorator for Jaseci Action interface"""
+    caller_globals = dict(inspect.getmembers(
+        inspect.stack()[1][0]))["f_globals"]
+    if allow_remote and 'serv_actions' not in caller_globals:
+        caller_globals['serv_actions'] = serv_actions
```
Comment (ypkang) [inline jaseci_core/jaseci/actions/live_actions.py:19]: I am curious what does this `caller_globals` do and why is this change conditioned if the action is allowed to be remote or not?

**Review by ypkang — CHANGES_REQUESTED** [review]
There is one issue with this change -- the unit tests for the jskit service rely on the `app` variable to function and it seems like in this version, the `app` variable is no longer available in each server's own python file.

File: jaseci_core/jaseci/actions/live_actions.py (hunk 1 above)
Comment (marsninja) [reply jaseci_core/jaseci/actions/live_actions.py:19]: Yea, you'd have to use that function serve_actions.

**Review by ypkang — APPROVED** [review]
LGTM.

**Pull request merged**